specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2023-02-01/vaults.json

```diff
@@ -1190,6 +1190,44 @@
         }
       }
     },
+    "SoftDeleteSettings": {
+      "description": "Soft delete Settings of vault",
+      "type": "object",
+      "properties": {
+        "softDeleteState": {
+          "enum": [
+            "Invalid",
+            "Enabled",
+            "Disabled",
+            "AlwaysON"
```

Comment:
nit, should it be capitalized 'AlwaysOn'? AlwaysON isn't the usual for PascalCase.

Reply:
(But by all means make it match your backends current behavior)

Reply:
this is as per our service